plugins/Diagnostics/Diagnostic/DiagnosticResult.php

```diff
@@ -0,0 +1,121 @@
+<?php
+/**
+ * Piwik - free/libre analytics platform
+ *
+ * @link http://piwik.org
+ * @license http://www.gnu.org/licenses/gpl-3.0.html GPL v3 or later
+ */
+
+namespace Piwik\Plugins\Diagnostics\Diagnostic;
+
+/**
+ * The result of a diagnostic.
```

Comment:
Ah ok, I assumed it was, maybe not permanent, but convention. Can you create a followup issue after this is merged about cleaning up diagnostic output (in terms of existing code + the new API)?